lib/dot.js

```diff
@@ -12,6 +12,9 @@ export default class Dot extends TileLayer.Canvas {
   initialize (grid, color = '#49a0d7') {
     this.color = new Color(color).rgbArray()
     this.grid = grid
+    //this.colorScale = new CutoffColorScale(1000, [0, 0, 0, 0], [...this.color, 128])
+    this.colorScale = new InterpolatingColorScale([[200, 0, 0, 200, 0], [2000, 0, 0, 200, 180], [20000, 150, 0, 0, 180]])
```

Comment:
If I understand this correctly, the dot map module now handles all types of maps that require interpolation. We should rename accordingly and add a ColorScale parameter. We should also refactor the existing choropleth code to use the same color scales, unless we decide to drop non-interpolated choropleths altogether.